modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -21,6 +21,13 @@ image::op-pipeline-list.png[]
 . Optionally, in the *Pipeline Details* page:
 
 * Click the *YAML* tab to edit the YAML file for the Pipeline.
-* Click the *Pipeline Runs* tab to see the completed, running, or failed runs for the Pipeline. You can use the Options menu {kebab} to stop a running Pipeline, to rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or to delete a PipelineRun.
+* Click the *Pipeline Runs* tab to see the completed, running, or failed runs for the Pipeline.
++
+[NOTE]
+====
+If a pipeline run has failed, the *Details* section of the *Pipeline Run Details* page displays a *Log Snippet* of the failed runs for the pipeline. *Log Snippet* provides a general error message and a snippet of the log, along with a link to the *Logs* section to quickly access the details about the failed run.
+The *Log Snippet* is also displayed on the the *Details* section of the *Task Run Details* page.
+====
+You can use the Options menu {kebab} to stop a running Pipeline, to rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or to delete a PipelineRun.
 * Click the *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
-* Click the *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
+* Click the *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
```

Comment:
All mentions of Pipeline can be converted to pipeline on line 31,32,33
Line 31 PipelineRun/pipeline run